### PR TITLE
Capture page apply responsivity

### PR DESCRIPTION
**Pull request by guilherme-gjv**

### CODE REVIEW

**Comment by guilherme-gjv** [comment]
## 2XL @media (min-width: 1536px)

image

## XL @media (min-width: 1280px)

image

## LG @media (min-width: 1024px)

image

## MD @media (min-width: 768px)

image

## SM @media (min-width: 640px)

image

## < SM

image

File: src/pages/captura.tsx [hunk 1]
```diff
@@ -189,18 +206,22 @@ const Captura = () => (
           <a href={linkMonetizzeMensal}>
             <button
               type="button"
-              className="uppercase xl:text-xl 2xl:text-[1.875rem]  mt-[12.75rem]"
-              style={twButton(100)}
+              className="uppercase absolute inset-x-0 -bottom-5 xl:text-xl 2xl:text-[1.875rem]"
```
Comment (LOFyz) [inline src/pages/captura.tsx:209]: Fix this `-` on the left of the class `bottom-5`

File: src/pages/captura.tsx [hunk 2]
```diff
@@ -189,18 +206,22 @@ const Captura = () => (
...
+              style={twButton(70)}
```
Comment (LOFyz) [inline src/pages/captura.tsx:210]: Remove this style prop. Make this only with class. Make it with [@apply](https://tailwindcss.com/docs/reusing-styles#extracting-classes-with-apply), create a plugin or make normally in className

File: src/pages/captura.tsx [hunk 3]
```diff
@@ -211,8 +232,8 @@ const Captura = () => (
           <a href={linkMonetizzeAnual}>
             <button
               type="button"
-              className="uppercase xl:text-xl 2xl:text-[1.875rem] xl:mt-[4.5rem] 2xl:mt-24"
-              style={twButton(100)}
+              className="uppercase absolute inset-x-0 -bottom-5 xl:text-xl 2xl:text-[1.875rem]"
+              style={twButton(70)}
```
Comment (LOFyz) [inline src/pages/captura.tsx:236]: Same as the other comment

File: src/pages/captura.tsx [hunk 4]
```diff
@@ -242,30 +263,32 @@ const Captura = () => (
...
+          não tinha ninguém para me orientar e auxiliar. Hoje represento um dos
+          maiores e melhores grupos do Brasil, tendo alunos que faturam na casa
+          dos 100 mil reais nos 4 cantos do país.
         </p>
-        <div className="w-1/2">
-          <img src="" alt="" />
-          <img src="" alt="" />
+        <div className="my-4 md:w-1/2 relative bg-red-400">
+          <img src="" alt="fotogb" />
+          <img src="" alt="fofo" />
```
Comment (LOFyz) [inline src/pages/captura.tsx:280]: Fix this alt when adding the image

**Review by LOFyz — CHANGES_REQUESTED** [review]
Solve my comments and request review again. This pull request if approved, can just be merged after the images being added.

File: src/pages/captura.tsx (hunk 1 above)
Comment (guilherme-gjv) [reply src/pages/captura.tsx:209]: The ``-`` is for negative bottom position

**Pull request merged**